# Data analysis side of proteomic research

This is a soft introduction to the data analysis side of proteomic research. These are interactive guides that walk you through the beginning step of proteomics. Our goal is to create an online interactive module that serves as a primer to the foundational concepts of mass spectrometry and proteomics. In many university settings, proteomics is taught from a chemistry perspective and lacks an interactive exploration of data sets. We approach proteomics from a data perspective to help train and attract bioinformatics and computational scientists into the field.

## sources used to within the notebooks.

Using publicly available data and interactive Python notebooks, we created a suite of tutorials to demonstrate the major data types within LC-MS/MS. We also utilize several public software packages to assist in graphing and data access (spectrum_utils, ppx, and pyteomics). 

### Tutorials
- [Lesson 1 : Measuring Mass](https://colab.research.google.com/drive/1ddDCRrI3jErP-2xPB2yiKt6pj1dmkvbu)
- [Lesson 2: Mass measurements of complex mixtures]([https://paynelab.github.io/cptac/tutorial02_pandas.html](https://colab.research.google.com/drive/1kgfcYbPHUdMcxbJ18Ija8hkqseHnC_e-#scrollTo=A8knw_sHCfwd))
- [Lesson 3: LC-MS]([https://paynelab.github.io/cptac/tutorial03_joining_dataframes.html](https://colab.research.google.com/drive/17KQ-9hWizhQyUAGp5V1KVJntKQ-RPAAt#scrollTo=aksoSpiLFaFd))
- [Lesson 4: LC-MS/MS](https://paynelab.github.io/cptac/tutorial04_multiindex.html)
